docs/pages/snapshots.md

```diff
@@ -5,12 +5,17 @@ This is not a problem if there are a few hundred.
 But if the number gets bigger at some point, then loading and rebuilding can become slow.
 The `snapshot` system can be used to control this.
 
-Normally, the events are all executed again on the aggregate in order to rebuild the current state.
+!!! note
+
+    In oure benchmarks we can load 10 000 events for one aggregate in 50ms.
```

Comment:
```suggestion
    In our benchmarks we can load 10 000 events for one aggregate in 50ms.
```

Generally im not sure if this note here is *really* helpful. Maybe add a separate section about performance with more context etc?